### PR TITLE
feat: implement `rocks purge` and `rocks remove`

### PR DESCRIPTION
**Pull request by vhyrro**
This PR adds basic support for `rocks purge` and `rocks remove`.

There are no tests for this yet, I'll get those set up tomorrow.

### CODE REVIEW

File: rocks-bin/src/purge.rs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+use eyre::Result;
+use rocks_lib::{config::Config, tree::Tree};
+
+pub fn purge(config: Config) -> Result<()> {
```
Comment (mrcjkb) [inline rocks-bin/src/purge.rs:4]: question: would it make sense to prompt for confirmation by default, and add a `--skip-prompt`  flag?

File: rocks-lib/src/tree/mod.rs [hunk 2]
```diff
@@ -50,6 +50,10 @@ impl Tree {
         self.root.join(self.version.to_string())
     }
 
+    pub fn root_for(&self, rock_name: &PackageName, rock_version: &PackageVersion) -> PathBuf {
```
Comment (mrcjkb) [inline rocks-lib/src/tree/mod.rs:53]: suggestion: why not take a `&LuaPackage` as an argument?

Comment (vhyrro) [reply rocks-lib/src/tree/mod.rs:53]: That's to keep consistency with the `rock()` function, which also takes a name and version. Those should be refactored to just use `LuaPackage` eventually though.

**Review by mrcjkb — APPROVED** [review]

**Comment by vhyrro** [comment]
Will merge this after #94

**Pull request merged**